app/api/dao/mentorship_relation.py

```diff
@@ -19,7 +20,9 @@ class MentorshipRelationDAO:
         MINIMUM_MENTORSHIP_DURATION
     """
 
-    MAXIMUM_MENTORSHIP_DURATION = timedelta(weeks=24)  # 6 months = approximately 6*4
+    MAXIMUM_MENTORSHIP_DURATION = timedelta(
+        weeks=24
+    )  # 6 months = approximately 6*4
```

Comment:
This comment is not depicting anything useful and can be removed

Reply:
Sure.